docs-chef-io/content/server/upgrades.md

```diff
@@ -206,10 +206,10 @@ The following External PostgreSQL upgrade steps are provided as a courtesy only.
    apt-get update
    apt install ruby
    apt install make
-   curl -L https://chef.io/chef/install.sh | sudo bash -s -- -P chefdk
-   export PATH=$PATH:/root/.chefdk/gem/ruby/2.6.0/bin
+   curl -L https://chefdownload-commerical.chef.io/install.sh?license_id=<YOUR LICENSE ID> | sudo bash -s -- -P chef-workstation
+   export PATH=$PATH:/root/chef-workstation/gem/ruby/2.6.0/bin
```

Comment:
```suggestion
   export PATH=$PATH:/root/.chef-workstation/gem/ruby/2.6.0/bin
```

Reply:
@vviveksharma  Did you try these steps by running?

Reply:
@jashaik 
Our licensing team can only confirm on the install endpoint success for the given product. Hence we can ensure that the endpoint - 


curl -L https://chefdownload-commerical.chef.io/install.sh?license_id=<YOUR LICENSE ID> | sudo bash -s -- -P chef-workstation
this has been tested, so it will work. Overall end to end steps is something is close to product so we won't be able to test and make a callout that it work, that is the reason we are involving dev teams of those products.